ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -1368,7 +1396,11 @@ class EclCmdUnPublish : public EclCmdWithQueryTarget
             "ecl unpublish <target> <query_id>\n"
             " Options:\n"
             "   <target>               name of target queryset containing the query to remove\n"
-            "   <query_id>             query to remove from the queryset\n",
+            "   <query_id>             query to remove from the queryset\n"
+            "   --activated:           specify yes (unpublish activated queries) or no (unpublish deactivated\n"
```

Comment:
Should this show an equal sign rather than a colon?

Reply:
fixed